### PR TITLE
Issue13 performance metric class

### PR DESCRIPTION
**Pull request by artuso0matteo**
The merging of this branch is a possible solution for what reported in [Issue 13](https://github.com/Sonohi/heyohletsgo/issues/13).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 root = true
 
 [*]
-indent_style = space
-indent_size = 4
+indent_style = tab
```
Comment (jakthra) [inline .editorconfig]: noooooooooooooooooooooooooooooooooooooooooooooo

File: enb/enbRxBulk.m [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
+%
+%   Stations	-> updated eNodeB objects
+
+for iStation = 1:length(Stations)
+	enb = Stations(iStation);
+	
+	% First off, check whether this station has received anything in UL.
+	% If not, it simply means that there are no UEs connected to it
+	if isempty(enb.Rx.Waveform)
+		sonohilog(sprintf('eNodeB %i has an empty received waveform',enb.NCellID), 'WRN');
```
Comment (jakthra) [inline enb/enbRxBulk.m]: Maybe reconsider 'WRN'?

File: results/MetricRecorder.m [hunk 3]
```diff
@@ -0,0 +1,168 @@
+classdef MetricRecorder
+	properties 
+		infoUtilLo;
+		infoUtilHi;
+		util;
+		power;
```
Comment (jakthra) [inline results/MetricRecorder.m]: is `power` explicit enough? Maybe `power_watt`

File: results/MetricRecorder.m [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+		schedule;
+		harqRtx;
+		arqRtx;
+		ber;
+		snr;
+		snrdB;
```
Comment (jakthra) [inline results/MetricRecorder.m]: Should we just keep one scale for power based values? I.e. either linear or db.

File: .editorconfig (hunk 1 above)
Comment (artuso0matteo) [reply .editorconfig]: https://www.youtube.com/watch?v=SsoOG6ZeyUI

File: enb/enbRxBulk.m (hunk 2 above)
Comment (artuso0matteo) [reply enb/enbRxBulk.m]: Yes, that shall be switched to INFO

File: results/MetricRecorder.m (hunk 3 above)
Comment (artuso0matteo) [reply results/MetricRecorder.m]: good suggestion!

**Pull request merged**